spring-batch-docs/modules/ROOT/pages/step/tasklet.adoc

```diff
@@ -122,7 +122,7 @@ public class FileDeletingTasklet implements Tasklet, InitializingBean {
     public RepeatStatus execute(StepContribution contribution,
                                 ChunkContext chunkContext) throws Exception {
         File dir = directory.getFile();
-        Assert.state(dir.isDirectory());
+        Assert.state(dir.isDirectory(), "Directory does not exist");
```

Comment:
Good catch! The message is missing and the code example does not compile. However, the message here is inaccurate, it would be appropriate if the assertion was `dir.exists()`. It should rather be something like "The resource must be a directory". Do you agree?

Reply:
Yes. I updated message. Thanks:)